src/Drips.sol

```diff
@@ -483,20 +472,12 @@ library Drips {
             } else if (pickCurr) {
                 // Remove an existing drip
                 mapping(uint32 => AmtDelta) storage deltas = states[currRecv.userId].amtDeltas;
-                (uint32 start, uint32 end) = _dripsRangeInFuture(
-                    currRecv,
-                    lastUpdate,
-                    currDefaultEnd
-                );
+                (uint32 start, uint32 end) = _dripsRangeInFuture(currRecv, currDefaultEnd);
```

Comment:
Also here the `lastUpdate` is not needed because the `startCap=currTimestamp` will overwrite it.